### ISSUE TITLE
Warehouse Window in WebUI Layout

### ISSUE THREAD

**Issue by metas-mk** [metasfresh/metasfresh#1178]
### Is this a bug or feature request?
Feature
### What is the current behavior?
The Warehouse Window has only the default layout until now.
#### Which are the steps to reproduce?
Open the window Warehouse and see.
### What is the expected or desired behavior?
Create the 2 column Layout for the Warehouse window and further Window elements (Grid Layout, Side List, Advanced Edit, Included Tabs).

**Comment by metas-rc** [comment]
Bugs found while testing this task: 
https://github.com/metasfresh/metasfresh-webui-api/issues/262
https://github.com/metasfresh/metasfresh-webui-api/issues/263

**Comment by metas-rc** [comment]
Another problem I found and I think we shall create  a task for it but I am not sure if it isn't already covered by other planning task we have: 
Hi. Do we have any task for showing messages about uniquie indexes when creating new data in webui? 
Context: I was playing with Warehouse, and when I pressed on Beanstandunglager flag ( isIssueWarehouse) I got an error in swing:
Caused by: org.postgresql.util.PSQLException: ERROR: duplicate key value violates unique constraint "isissuewarehouse_uniqueindex"
But when I do the same in webui it says nothing, only the red line appears. And then, if I want to create a line, I get:
Image Pasted at 2017-3-20 18-27.png
PNG252KB
6:28 PM
there is no info about the reason why the entry is invalid
